docs/_docs/starting-nodes.adoc

```diff
@@ -213,6 +213,28 @@ You can also pass JVM options by using the `-J` prefix:
 ----
 ./ignite.sh -J-Xmx6G -J-DIGNITE_TO_STRING_INCLUDE_SENSITIVE=false
 ----
+
+== Ignite Internal Settings
+Ignite provide more then hundred internal flags to tweak som apects of the node behavior.
+You can see all flags, their descriptions and default values with the following command:
+
+[source, shell]
+----
+./ignite.sh -systemProps
+----
+
+Example of the output:
+
+[source, text]
+$ ./bin/ignite.sh -systemProps
+IGNITE_AFFINITY_HISTORY_SIZE                                    - [Integer] Maximum size for affinity assignment history. Default is 25.
```

Comment:
Please put digits, constant names or bool values into `value` brackets so that it's highlighted with a special style. For instance, in this row you should change 25 to `25`. Do for all similar places.

Reply:
But this is just an example of console output.
There is no formatting in the console.